scripts/java-schema.json

```diff
@@ -0,0 +1,167 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "common": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "unique id"
+                },
+                "description": {
+                    "type": "string",
+                    "description": "human-readable description"
+                },
+                "operation": {
+                    "type": "string",
+                    "description": "human-readable description"
+                },
+                "start": {
+                    "type": "integer",
+                    "description": "start time for test (in frames)"
+                }
+            },
+            "required": [
+                "id",
+                "description"
+            ],
+            "additionalProperties": false
+        },
+
+        "input": {
+            "type": "object",
+            "properties": {
+                "filepath": {
+                    "type": "string",
+                    "description": "full file path at the DUT"
+                },
+                "width": {
+                    "type": "integer",
+                    "description": "file width"
+                },
+                "height": {
+                    "type": "integer",
+                    "description": "file height"
+                },
+                "pix_fmt": {
+                    "type": "string",
+                    "description": "file pixel format (ffmpeg format)"
+                },
+                "framerate": {
+                    "type": "integer",
+                    "description": "file frame rate (fps)"
+                },
+                "duration": {
+                    "type": "integer",
+                    "description": "duration of the file (frames)"
+                },
+                "drops": {
+                    "type": "array",
+                    "description": "list of frames to be dropped (number)",
+                    "items": {
+                        "type": "number"
+                    },
+                    "minItems": 1,
+                    "uniqueItems": true
+                },
+                "dynamic-framerate": {
+                    "type": "array",
+                    "description": "list of frames to be dropped (number)",
```

Comment:
Nitpick. This is start position of new frame rate. The runtime time calculation will determine which frames will be dropped (or duplicated).